index.html

```diff
@@ -323,113 +246,59 @@ <h3 class="card__title">Skills</h3>
           </div>
         </div>
       </div>
+      <div class="about-me__bg">
+        <img src="./dist/images/about-bg.svg" alt="ksdfj" />
+      </div>
     </section>
     <section id="contact-form">
       <div class="wrapper">
         <div class="contact-form__inner">
-          <h2 class="contact-form__title title--3">
-            I'm always interested in hearing about new projects, so if you'd
-            like to chat please get in touch.
-          </h2>
-          <form
-            action="https://formspree.io/f/myyvbjla"
-            method="POST"
-            class="contact-form__form"
-          >
-            <input
-              type="text"
-              id="contact-form__full-name"
-              placeholder="Full name"
-              name="full-name"
-              maxlength="30"
-              required
-            />
-            <input
-              type="email"
-              placeholder="Email address"
-              id="contact-form__email"
-              name="email"
-              required
-            />
-            <textarea
-              name="message"
-              id="contact-form__message"
-              cols="30"
-              rows="5"
-              placeholder="Enter text here"
-              maxlength="500"
-              required
-            ></textarea>
-            <button class="btn btn--primary">Get in touch</button>
+          <h2 class="contact-form__title title--3">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+          <form action="https://formspree.io/f/myyvbjla" method="POST" class="contact-form__form">
+            <input type="text" id="contact-form__full-name" placeholder="Full name" name="full-name" maxlength="30" required />
+            <input type="email" placeholder="Email address" id="contact-form__email" name="email" required />
+            <textarea name="message" id="contact-form__message" cols="30" rows="5" placeholder="Enter text here" maxlength="500" required></textarea>
+            <div class="contact-form__btn-wrapper">
+              <button class="btn btn--primary">Get in touch</button>
```

Comment:
- [ ] The form button is **center-aligned for mobile screens** and **left-aligned for desktop screens** 📝, but in your design, it's **left-aligned** for both `mobile` and `desktop` screens. Kindly fix this 👇🏽:

### Your design:
![image](https://user-images.githubusercontent.com/43172164/200039987-ad68bc12-f98b-48c2-9d3c-8e6e3a8d8905.png)

### Figma design:
![image](https://user-images.githubusercontent.com/43172164/200040136-88ce7781-8b8d-47bb-837e-cee95f1e2f96.png)